cookbooks/aws-parallelcluster-platform/templates/pyxis/pyxis.conf.erb

```diff
@@ -0,0 +1 @@
+ runtime_path=<%= node['cluster']['shared_dir'] %>/pyxis/
```

Comment:
the pyxis.conf file should contain this content:
```
required /usr/local/lib/slurm/spank_pyxis.so 
runtime_path=/opt/parallelcluster/shared/pyxis
```

Reply:
Thank you was just creating a cluster to check the contents